src/webcomponents/variant/interpretation/variant-interpreter-review.js

```diff
@@ -245,164 +177,128 @@ export default class VariantInterpreterReview extends LitElement {
-                                            .clinicalVariants="${variants || []}"
-                                            .active="${active}"
-                                            .toolId="${"variant-interpreter-rearrangement"}"
-                                            .gridConfig="${gridConfig}"
-                                            .settings="${this.settings.browsers["REARRANGEMENT"]}">
-                                        </variant-interpreter-review-primary>
-                                    ` : html`
-                                        <div class="alert alert-info">
-                                            No <b>Somatic Rearrangements</b> to display.
-                                        </div>
-                                    `}
-                                </div>
+                                ${variants.length > 0 ? html`
+                                    <h3>Somatic Rearrangements</h3>
+                                    <variant-interpreter-rearrangement-grid
+                                        .toolId="${"variant-interpreter-rearrangement"}"
+                                        .opencgaSession="${opencgaSession}"
+                                        .clinicalAnalysis="${clinicalAnalysis}"
+                                        .clinicalVariants="${variants}"
+                                        .review="${true}"
+                                        .active="${active}"
+                                        .config="${gridConfig}">
+                                    </variant-interpreter-rearrangement-grid>
+                                ` : nothing}
                             `;
                         },
                     });
                 }
 
                 if (germlineSample) {
-                    // Add Germline Small Variants tab
                     items.push({
                         id: "germline-small-variants",
-                        name: "Germline Small Variants",
-                        render: (clinicalAnalysis, active, opencgaSession) => {
-                            const variants = clinicalAnalysis?.interpretation?.primaryFindings
+                        render: (clinicalAnalysis, allVariants, active, opencgaSession) => {
+                            const variants = (allVariants || [])
                                 ?.filter(v => v.studies[0]?.samples[0]?.sampleId === germlineSample?.id)
                                 ?.filter(v => v.type !== "BREAKEND");
                             const gridConfig = {
+                                ...defaultGridConfig,
+                                ...(this.settings?.browsers?.["RD"]?.table || {}),
+                                ...(this.gridConfig || {}),
                                 somatic: false,
                                 variantTypes: ["SNV", "INDEL", "INSERTION", "DELETION"],
```

Comment:
Similar to the comment above– double-check with bioinfo when possible. Could be MNV considered germline small variant?